README.md

```diff
@@ -159,6 +160,14 @@ If you have had to expose Bono and Ellis in a non-standard manner, you may need
 rake test[default.svc.cluster.local] PROXY={{Bono external IP addresss}} ELLIS={{external IP address of one of your nodes}}:30080 SIGNUP_CODE=secret
 ```
 
+### Scaling the deployment
+
+All of the stateless Clearwater services: i.e. everything except Cassandra, Chronos and Astaire can be dynamically scaled up and down by running e.g. 
+`kubectl scale deployment sprout --replicas=3`
+(The other exception is Bono depending on how it is exposed external to your deployment -- see above).
```

Comment:
We're saying "(see limitations above)" way too many times here. Just needs a bit of rewording for readability. 
"Other than the limitations stated in the 'limitations' section, all our stateless services can be scaled up and down; For stateful services, there are some further restrictions, but in general all can be scaled up, and only cassandra can't be scaled down."?